src/test/java/hudson/remoting/PrefetchingTest.java

```diff
@@ -40,9 +41,7 @@ protected void setUp() throws Exception {
                 new URL[] {toFile(jar1).toURI().toURL(), toFile(jar2).toURI().toURL()},
                 this.getClass().getClassLoader());
 
-        dir = File.createTempFile("remoting", "cache");
-        dir.delete();
-        dir.mkdirs();
+        dir = Files.createTempDirectory("remoting" + "cache").toFile();
```

Comment:
The concatenation of strings "remoting" + "cache" looks odd to me. I think a single string would be more concise. And if we are to use a single string, might as well make the value more readable by using a separator, e.g.:

```java
        dir = Files.createTempDirectory("remoting-cache").toFile();
```

Reply:
I'm inclined to agree, that would be ideal. Feel free to use suggested changes to commit that change on this branch